src/libstd/macros.rs

```diff
@@ -98,7 +98,9 @@ macro_rules! print {
     ($($arg:tt)*) => ($crate::io::_print(format_args!($($arg)*)));
 }
 
-/// Macro for printing to the standard output, with a newline.
+/// Macro for printing to the standard output, with a newline. On all
```

Comment:
Colloquially to me I think "On all platforms" makes a little more sense than "on every platforms"